bundles/config/org.eclipse.smarthome.config.discovery/src/main/java/org/eclipse/smarthome/config/discovery/internal/DiscoveryResultImpl.java

```diff
@@ -49,6 +51,7 @@
      * @param label the human readable label to set (could be null or empty)
      * @param bridgeUID the unique bridge ID to be set
      * @param timeToLive time to live in seconds
+     * @param discoveredBy class name of the {@link DiscoveryService}, that created this result
```

Comment:
I assume you have to make the parameter a string (and not keep it a Class for having strong types) due to the JSON (de)serialization, right? If so, you might want to add "fqcn", so that it is clear that it is not the simple class name that is excepted here.